src/NUnitTestAdapter/NUnit3TestExecutor.cs

```diff
@@ -264,7 +264,7 @@ private void RunAssembly(string assemblyPath, TestFilter filter)
                             return;
                         }
 
-                        using (var listener = new NUnitEventListener(FrameworkHandle, testConverter, dumpXml))
+                        using (var listener = new NUnitEventListener(FrameworkHandle, testConverter, dumpXml, new AdditionalTestInfo()))
```

Comment:
Creates a new instance of additional test storage and puts it to the ctor

Reply:
Any reason you cant add this to the NUnitEventListener class ?   We try to keep the number of parameters down, if we can.

Reply:
Done. For example to make it possible to test NUnitEventListener in isolation. I prefer to inject dependencies